otter/models/cass.py

```diff
@@ -641,7 +649,12 @@ def wrapper(*args):
             return d
         return wrapper
 
-    def view_manifest(self, with_policies=True, with_webhooks=False):
+    def _group_status(self, status, deleting):
```

Comment:
Do we need to handle the case where the status is `None` or an invalid value?  There seem to be some `DISABLED` statuses currently in the DB.